okxchain/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "okxchain",
+  "assets": [
+    {
+      "description": "EVM compatible, Cosmos SDK based chain.",
+      "denom_units": [
+        {
+          "denom": "aokt",
+          "exponent": 0
+        },
+        {
+          "denom": "okt",
+          "exponent": 18
+        }
+      ],
+      "base": "aokt",
+      "name": "OKXChain",
+      "display": "OKT",
+      "symbol": "OKT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/evmos/images/okc.svg",
```

Comment:
-shouldn't be evmos' images folder.
-no image named 'okc', please upload a token logo image.
-the images uploaded so far look more like chain logo, being rectangular and having lettering. Please upload a token logo, where the file has a square aspect ratio, and that can still look good when cropped to an inner circle.